web/src/containers/Login.js

```diff
@@ -36,50 +37,52 @@ class Login extends Component {
 
     return (
       <Fragment>
-        <header className="clearfix px2 py1 bg-white">
-          <div className="left">
-            <Link to="/" className="btn px0 bold caps">
-              {t('titleBasic')}
-            </Link>
-          </div>
-        </header>
-
-        <div className="mx-auto my3 p2 sm-col-6 md-col-4 bg-white rounded">
-          <h1 className="mt0 h2">Please log in.</h1>
-          {error && (
-            <div className="mb2 p1 h6 alert alert-error">
-              <strong>Sorry!</strong> Something went wrong. Please try again.
-            </div>
-          )}
-          <form onSubmit={this.handleSubmit}>
-            <div className="mb2">
-              <label htmlFor="username">Email</label>
-              <input
-                id="username"
-                type="text"
-                name="username"
-                className="input"
-                value={username}
-                onChange={this.handleChange}
-              />
+        <Header />
+        <div className="card--container">
```

Comment:
This adds the gray background. I'd really rather add a body class to the page that colors the whole page background gray because it would be more robust, but figuring out how to bubble that up seemed hard. :-p